### DISCUSSION TITLE
划词翻译的会员功能可以改为一次购买永久使用吗？

### DISCUSSION THREAD

**Discussion by Koitoouav** [hcfyapp/crx-selection-translate#964] [讨论]
## 为什么你觉得需要（改进）这个功能？
不喜欢年付或月付的付费形式，建议出一个永久版的。

**Comment by lmk123** [comment]
我确实有计划将除了数据同步之外的功能出一个永久版的。等我先完成 v7.5.0 的开发吧

**Comment by lmk123** [comment]
最近关于这个问题我又有了一些想法，决定不暂时不改成永久版的了。

当我作为一个消费者去购买别人的服务的时候，我也希望能一次付费永久使用，但现在作为一个服务提供方，我更希望使用订阅制（也就是月付 / 年付的形式），原因如下：

- 我在持续付出我的时间和精力，并不是开发完一个版本之后就不再更新了。使用订阅制会让我有更多持续更新下去的动力——不断改进划词翻译、不断提供新的功能——这样才能吸引消费者继续订阅，消费者也能不断的获得新的使用体验，而我也可以持续获得一定的收入。如果一次付费永久使用，我可能就不会有这样的动力了，划词翻译大概率会停留在一个固定的版本中，只会在有重大 bug 出现的时候才修复一下。
- “永久”这个时间跨度太长了，划词翻译我已经维护了 7 年，但以后的事情谁也说不准，我也不能百分百确定我可以一直维护下去。如果我给消费者提供了“永久使用”的承诺，却在某一天没有继续维护下去，我觉得这是对消费者的不负责任。
- 我依赖的其它服务使用的也是订阅制。支撑划词翻译会员功能的云服务器是每年都要续费的；开发划词翻译使用的代码编辑器也是需要每年付费的；未来如果要上架 Safari 版划词翻译，App store 会[每年向我收取 99 美元](https://developer.apple.com/cn/support/purchase-activation/)的费用。

我觉得这个可以作为一个讨论帖，看看大家的想法如何。

**Comment by Koitoouav** [comment]
对于我而言，这种功能性的软件如果不出永久版的话，我大概率是不会为其付费的，因为付费的功能并不是刚需，相信有许多付费用户只是捐赠，而不是因功能而付费。我本身并不喜欢月付季付年付形式，并且说句实在的，这种工具付费只是给作者更多一份动力，使用场景并不多，我也没有移动端使用的习惯，在有永久的情况下提供月付季付年付不同形式，可以同时兼容大多数人群，在保证作者有持续更新动力的情况下也不会流失喜欢买断制付费的用户。

**Comment by Koitoouav** [comment]
> 最近关于这个问题我又有了一些想法，决定不暂时不改成永久版的了。
> 
> 当我作为一个消费者去购买别人的服务的时候，我也希望能一次付费永久使用，但现在作为一个服务提供方，我更希望使用订阅制（也就是月付 / 年付的形式），原因如下：
> 
> * 我在持续付出我的时间和精力，并不是开发完一个版本之后就不再更新了。使用订阅制会让我有更多持续更新下去的动力——不断改进划词翻译、不断提供新的功能——这样才能吸引消费者继续订阅，消费者也能不断的获得新的使用体验，而我也可以持续获得一定的收入。如果一次付费永久使用，我可能就不会有这样的动力了，划词翻译大概率会停留在一个固定的版本中，只会在有重大 bug 出现的时候才修复一下。
> * “永久”这个时间跨度太长了，划词翻译我已经维护了 7 年，但以后的事情谁也说不准，我也不能百分百确定我可以一直维护下去。如果我给消费者提供了“永久使用”的承诺，却在某一天没有继续维护下去，我觉得这是对消费者的不负责任。
> * 我依赖的其它服务使用的也是订阅制。支撑划词翻译会员功能的云服务器是每年都要续费的；开发划词翻译使用的代码编辑器也是需要每年付费的；未来如果要上架 Safari 版划词翻译，App store 会[每年向我收取 99 美元](https://developer.apple.com/cn/support/purchase-activation/)的费用。
> 
> 我觉得这个可以作为一个讨论帖，看看大家的想法如何。

当然，买断制的付费价格也要根据年付所定价格去判断，况且你所依赖的其它服务对你来说也许属于刚需，作为服务提供方只要你对这个软件的需求，他们当然会要求月付季付年付形式，同时你也强调**支撑划词翻译会员功能的云服务器是每年都要续费的**；这对你而言必然是刚需中的刚需，而这款软件对于我这种类型的用户来说并不是刚需，市场上也有相对应的替代品，但你的这个软件做的比其他同类软件更加优秀，这是我为什么想要买断制的原因，对于我而言，这是一种捐赠，并且享受这种服务，形式大于意义。

**Comment by Koitoouav** [comment]
> 最近关于这个问题我又有了一些想法，决定不暂时不改成永久版的了。
> 
> 当我作为一个消费者去购买别人的服务的时候，我也希望能一次付费永久使用，但现在作为一个服务提供方，我更希望使用订阅制（也就是月付 / 年付的形式），原因如下：
> 
> * 我在持续付出我的时间和精力，并不是开发完一个版本之后就不再更新了。使用订阅制会让我有更多持续更新下去的动力——不断改进划词翻译、不断提供新的功能——这样才能吸引消费者继续订阅，消费者也能不断的获得新的使用体验，而我也可以持续获得一定的收入。如果一次付费永久使用，我可能就不会有这样的动力了，划词翻译大概率会停留在一个固定的版本中，只会在有重大 bug 出现的时候才修复一下。
> * “永久”这个时间跨度太长了，划词翻译我已经维护了 7 年，但以后的事情谁也说不准，我也不能百分百确定我可以一直维护下去。如果我给消费者提供了“永久使用”的承诺，却在某一天没有继续维护下去，我觉得这是对消费者的不负责任。
> * 我依赖的其它服务使用的也是订阅制。支撑划词翻译会员功能的云服务器是每年都要续费的；开发划词翻译使用的代码编辑器也是需要每年付费的；未来如果要上架 Safari 版划词翻译，App store 会[每年向我收取 99 美元](https://developer.apple.com/cn/support/purchase-activation/)的费用。
> 
> 我觉得这个可以作为一个讨论帖，看看大家的想法如何。

也许你会告知说我们你不想出买断制服务，并且建议我们这些不喜欢付费形式的用户可以无需付费，仅使用基础功能，但这显然是与你的收益背道而驰的。所以何不选择提供多种选择，更加灵活，更加兼容不同使用的用户，买断制和订阅制并不冲突，也并不是所有用户都会选择买断制，甚至如果多出买断制，也许会在短期内给你增加更多客观的收入，这又何尝不是一件好事。

**Comment by lmk123** [comment]
如果划词翻译推出买断制，你觉得定价多少比较合适？

**Comment by Koitoouav** [comment]
> 如果划词翻译推出买断制，你觉得定价多少比较合适？

我认为应该是当前**年订阅价格 * 5 倍**，最起码付出五年或以上的订阅费，后期可通过功能性的更新来调整价格，功能越多，价格也会随之水涨船高。

**Comment by lmk123** [comment]
你说服我了，我会在近期添加永久会员的购买选项。

**Comment by korrasamiii** [comment]
> > 最近关于这个问题我又有了一些想法，决定不暂时不改成永久版的了。
> > 当我作为一个消费者去购买别人的服务的时候，我也希望能一次付费永久使用，但现在作为一个服务提供方，我更希望使用订阅制（也就是月付 / 年付的形式），原因如下：
> > 
> > * 我在持续付出我的时间和精力，并不是开发完一个版本之后就不再更新了。使用订阅制会让我有更多持续更新下去的动力——不断改进划词翻译、不断提供新的功能——这样才能吸引消费者继续订阅，消费者也能不断的获得新的使用体验，而我也可以持续获得一定的收入。如果一次付费永久使用，我可能就不会有这样的动力了，划词翻译大概率会停留在一个固定的版本中，只会在有重大 bug 出现的时候才修复一下。
> > * “永久”这个时间跨度太长了，划词翻译我已经维护了 7 年，但以后的事情谁也说不准，我也不能百分百确定我可以一直维护下去。如果我给消费者提供了“永久使用”的承诺，却在某一天没有继续维护下去，我觉得这是对消费者的不负责任。
> > * 我依赖的其它服务使用的也是订阅制。支撑划词翻译会员功能的云服务器是每年都要续费的；开发划词翻译使用的代码编辑器也是需要每年付费的；未来如果要上架 Safari 版划词翻译，App store 会[每年向我收取 99 美元](https://developer.apple.com/cn/support/purchase-activation/)的费用。
> > 
> > 我觉得这个可以作为一个讨论帖，看看大家的想法如何。
> 
> 当然，买断制的付费价格也要根据年付所定价格去判断，况且你所依赖的其它服务对你来说也许属于刚需，作为服务提供方只要你对这个软件的需求，他们当然会要求月付季付年付形式，同时你也强调**支撑划词翻译会员功能的云服务器是每年都要续费的**；这对你而言必然是刚需中的刚需，而这款软件对于我这种类型的用户来说并不是刚需，市场上也有相对应的替代品，但你的这个软件做的比其他同类软件更加优秀，这是我为什么想要买断制的原因，对于我而言，这是一种捐赠，并且享受这种服务，形式大于意义。

希望能够尽快上架ios，实现我在ipad上也能划词翻译，用过iOS上其他的跨软件翻译，效果始终不尽如意，还是喜欢划词翻译

**Comment by lmk123** [comment]
@korrasamiiiii3 扩展程序目前不能安装在 iOS 系统（iPad、iPhone）的 Safari 上，我上架了 App store

**Comment by lmk123** [comment]
@Dionysiakos 我将划词翻译的永久会员从 188 元降到了 98 元，如果你之前购买过永久会员，那么你应该收到了一封邮件，你将支付宝账号或微信收款码回复给这封邮件，我会将差额 90 元退给你。

**Comment by Koitoouav** [comment]
> @Dionysiakos 我将划词翻译的永久会员从 188 元降到了 98 元，如果你之前购买过永久会员，那么你应该收到了一封邮件，你将支付宝账号或微信收款码回复给这封邮件，我会将差额 90 元退给你。

啊? 出永久版了?这就买。

**Comment by Koitoouav** [comment]
![](https://i.imgur.com/iFQCrGW.png)

**Comment by lmk123** [comment]
我以为之前那个永久会员是你，就给你发了评论提醒你退款 :joy:

谢谢支持

**Comment by soulmz** [comment]
> 我以为之前那个永久会员是你，就给你发了评论提醒你退款 😂
> 
> 谢谢支持

@lmk123 已买哈，白嫖了你这么久，还是你的插件最好用，真的！

用过这么多划词翻译，就你最好用了。买断了，你可以以买断版本 形式提供 后续的收益。参考 surge 

但是如果需要提供云服务的话，订阅制肯定是比较好的。毕竟靠自己长期支付云服务费用，确实不可取。

但是，我依然希望你保持现状，哪怕采取**大版本买断形式，相信大部分人也是愿意付费。**

**Comment by lmk123** [comment]
@soulmz 感谢支持。目前先保持现状吧，以后再看云服务费用等多方的成本考虑变动。
